src/app/config.cc

```diff
@@ -74,34 +74,38 @@ void Configuration::ParseKovriConfig() {
   // See: i2p.i2p/router/java/src/net/i2p/router/transport/udp/UDPEndpoint.java
   // TODO(unassigned): move this elsewhere (outside of ParseArgs()) when possible
   std::uint16_t port = kovri::core::RandInRange32(9111, 30777);
-  // Configuration files
-  std::string kovri_config, tunnels_config;
   // Default visible option
   bpo::options_description help("\nhelp");
   help.add_options()("help,h", "");  // Blank so we can use custom message above
   // Map options values from command-line and config
   bpo::options_description system("\nsystem");
-  system.add_options()
-    ("host", bpo::value<std::string>()->default_value("127.0.0.1"))
-    ("port,p", bpo::value<int>()->default_value(port))
-    ("daemon,d", bpo::value<bool>()->default_value(false))
-    ("service,s", bpo::value<std::string>()->default_value(""))
-    ("log-to-console", bpo::value<bool>()->default_value(true))
-    ("log-to-file", bpo::value<bool>()->default_value(true))
-    ("log-file-name", bpo::value<std::string>()->default_value(
-        (kovri::core::GetLogsPath() / "kovri_%Y-%m-%d.log").string()))  // TODO(anonimal): use only 1 log file?
-    // Log levels
-    // 0 = fatal
-    // 1 = error fatal
-    // 2 = warn error fatal
-    // 3 = info warn error fatal
-    // 4 = debug info warn error fatal
-    // 5 = trace debug info warn error fatal
-    ("log-level", bpo::value<std::uint16_t>()->default_value(3))
-    ("kovriconf,c", bpo::value<std::string>(&kovri_config)->default_value(
-        (kovri::core::GetConfigPath() / "kovri.conf").string()))
-    ("tunnelsconf,t", bpo::value<std::string>(&tunnels_config)->default_value(
-        (kovri::core::GetConfigPath() / "tunnels.conf").string()));
+  system.add_options()(
+      "host", bpo::value<std::string>()->default_value("127.0.0.1"))(
+      "port,p", bpo::value<int>()->default_value(port))(
+      "data-dir",
+      bpo::value<std::string>()->default_value(
+          kovri::core::GetDefaultDataPath().string()))(
+      "daemon,d", bpo::value<bool>()->default_value(false))(
+      "service,s", bpo::value<std::string>()->default_value(""))(
+      "log-to-console", bpo::value<bool>()->default_value(true))(
+      "log-to-file", bpo::value<bool>()->default_value(true))(
+      "log-file-name",
+      bpo::value<std::string>()->default_value(""),
+      "$DATA-DIR/logs/kovri_%Y-%m-%d.log")  // TODO(anonimal): use only 1 log file?
+      // Log levels
+      // 0 = fatal
+      // 1 = error fatal
+      // 2 = warn error fatal
+      // 3 = info warn error fatal
+      // 4 = debug info warn error fatal
+      // 5 = trace debug info warn error fatal
+      ("log-level", bpo::value<std::uint16_t>()->default_value(3))(
+          "kovriconf,c",
+          bpo::value<std::string>()->default_value(""),
+          "$DATA-DIR/config/kovri.conf")(
+          "tunnelsconf,t",
+          bpo::value<std::string>()->default_value(""),
+          "$DATA-DIR/config/tunnels.conf");
```

Comment:
If you don't want to remove these `$DATA-DIR` placeholders, I'll end up doing so after this is merged. Just let me know.